ng-dev/pr/merge/strategies/autosquash-merge.ts

```diff
@@ -93,11 +93,11 @@ export class AutosquashMergeStrategy extends MergeStrategy {
 
     this.pushTargetBranchesUpstream(targetBranches);
 
-    // For PRs which do not target the `master` branch on Github, Github does not automatically
-    // close the PR when its commit is pushed into the repository.  To ensure these PRs are
+    // For PRs which do not target the main branch on Github, Github does not automatically
+    // close the PR when its commit is pushed into the repository. To ensure these PRs are
     // correctly marked as closed, we must detect this situation and close the PR via the API after
     // the upstream pushes are completed.
-    if (githubTargetBranch !== 'master') {
+    if (githubTargetBranch !== this.git.mainBranchName) {
```

Comment:
```suggestion
    if (githubTargetBranch !== getNextBranchName(this.git.mainBranchName)) {
```

Reply:
As discussed offline. This should be different to the next branch, but rather about the main branch set for the repository. Could be different to the `next` release-train branch